storage/volume/driver/proxy.go

```diff
@@ -0,0 +1,173 @@
+package driver
+
+import (
+	"errors"
+
+	"github.com/alibaba/pouch/plugins"
+)
+
+const (
+	remoteVolumeCreateService       = "/VolumeDriver.Create"
+	remoteVolumeRemoveService       = "/VolumeDriver.Remove"
+	remoteVolumeMountService        = "/VolumeDriver.Mount"
+	remoteVolumeUnmountService      = "/VolumeDriver.Unmount"
+	remoteVolumePathService         = "/VolumeDriver.Path"
+	remoteVolumeGetService          = "/VolumeDriver.Get"
+	remoteVolumeListService         = "/VolumeDriver.List"
+	remoteVolumeCapabilitiesService = "/VolumeDriver.Capabilities"
+)
+
+// remoteDriverProxy is a remote driver proxy.
+type remoteDriverProxy struct {
+	Name   string
+	client *plugins.PluginClient
+}
+
+// Create creates a volume.
+func (proxy *remoteDriverProxy) Create(name string, opts map[string]string) error {
+	var req = remoteVolumeCreateReq{
+		Name: name,
+		Opts: opts,
+	}
+
+	var resp remoteVolumeCreateResp
+
+	if err := proxy.client.CallService(remoteVolumeCreateService, &req, &resp, true); err != nil {
```

Comment:
I am wondering if we could change the function, just making the response in the returned value, instead of passing a pointer of resp in parameter? Does it to meed some particular demand?

Reply:
To simplize the caller logic. Return io.Reader?